samples/micrometer-samples-kotlin/build.gradle

```diff
@@ -23,6 +23,8 @@ dependencies {
 
     testImplementation project(":micrometer-test")
 
+    testImplementation platform(libs.junitBom)
+    testRuntimeOnly platform(libs.junitBom)
```

Comment:
We shouldn't need to declare the bom at testRuntimeOnly scope but the `downloadDependencies` task failed if I didn't here. I suspect it must have something to do with Kotlin since the other projects didn't need the same.